slacktest/types.go

```diff
@@ -64,7 +63,6 @@ type Server struct {
 	registered           map[string]struct{}
 	server               *httptest.Server
 	mux                  *http.ServeMux
-	Logger               *log.Logger
```

Comment:
I wouldn't remove this - unnecessary breaking change imo.

And in future, it would actually be nice to use `sts.Logger` to log instead of importing log and logging directly. So let's keep it.